server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -161,6 +162,51 @@ const accountMutations = {
       return account;
     },
   },
+  removeTwoFactorAuthTokenFromIndividual: {
+    type: new GraphQLNonNull(Individual),
+    description: 'Add 2FA to the Account if it does not have it',
```

Comment:
@sbinlondon also wrong description

Reply:
Ok have popped it onto this PR https://github.com/opencollective/opencollective-api/pull/4946